snuba/cli/devserver.py

```diff
@@ -215,6 +215,30 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                     "--consumer-group=snuba-metrics-consumers",
                 ],
             ),
+            (
```

Comment:
Do you want to use the same settings flag we use for release health metrics or use a different flag `settings.ENABLE_SENTRY_METRICS_DEV`? Will there be a use case where someone wants to run just one of release health metrics vs performance metrics?

To enable the indexer in Sentry for release health vs performance metrics, is there a single setting or different settings? Maybe it might be a good idea to keep it consistent.

Reply:
I did in fact copy the single-setting behavior from Sentry (Markus added this recently to help local e2e testing): https://github.com/getsentry/sentry/pull/36682